README.md

```diff
@@ -446,6 +446,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Laravel'de Unit Test Kullanımı](https://kodumunblogu.net/detail/laravelde-unit-test-kullanimi)
 - [Laravel'de Transaction Kullanımı](https://kodumunblogu.net/detail/laravelde-transaction-kullanimi)
 - [Laravel'de Swagger ile Rest Api Dökümantasyonu Hazırlama](https://kodumunblogu.net/detail/laravelde-swagger-ile-rest-api-dokumantasyonu-hazirlama)
+- [Laravel Mix Kullanımı](https://www.oktaybozaci.com/html-css/laravel-mix-kullanimi.html)
```

Comment:
`npm install` oncesinde projeyi indirsek/klonlasak daha acik olurmus sanki. rastgele install yapmaya calisiyoruz gibi hissettim okurken.

Reply:
Dönüşünüz için teşekkürler. Gerekli düzenlemeleri yapacağım.